src/applications/vaos/tests/appointment-list/components/AppointmentsPage/index.unit.spec.jsx

```diff
@@ -921,4 +922,28 @@ describe('VAOS integration: appointment list', () => {
     ).to.be.ok;
     expect(screen.getByRole('link', { name: 'Request an appointment' }));
   });
+
+  it('should show COVID-19 appt schedule button', async () => {
+    const defaultState = {
+      featureToggles: {
+        ...initialState.featureToggles,
+        vaOnlineSchedulingCheetah: true,
+      },
+      user: userState,
+    };
+    const screen = renderWithStoreAndRouter(<ScheduleNewProjectCheetah />, {
```

Comment:
I think it would be better in this test to render `<AppointmentsPage />`, like the test above does. That way you can test that the feature toggle check works.

Reply:
Done, replaced the render with <AppointmentsPage />